### ISSUE TITLE
Licecap installation fails: ` Directory not empty @ dir_s_rmdir - /var/folders/4d/d_nhdwh14m1f6pf07417q4kw0000gn/T/d20180723-49512-zd6fma`

### ISSUE THREAD

**Issue by Taytay** [Homebrew/homebrew-cask#49929]
I did not see this problem listed. If I missed the template, feel free to close.

I just installed Home-brew tonight. Other casks are working great. 
This is the error I see:
```
Installing licecap cask. It is not currently installed.
==> Satisfying dependencies
==> Downloading https://www.cockos.com/licecap/licecap128.dmg
Already downloaded: /Users/taytay/Library/Caches/Homebrew/Cask/licecap--1.28.dmg
==> Verifying checksum for Cask licecap
==> Installing Cask licecap
==> Purging files for version 1.28 of Cask licecap
Error: Directory not empty @ dir_s_rmdir - /var/folders/4d/d_nhdwh14m1f6pf07417q4kw0000gn/T/d20180723-49512-zd6fma
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1446:in `rmdir'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1446:in `block in remove_dir1'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1457:in `platform_support'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1445:in `remove_dir1'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1438:in `remove'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:780:in `block in remove_entry'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1495:in `ensure in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1495:in `postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:778:in `remove_entry'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/tmpdir.rb:95:in `ensure in mktmpdir'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/tmpdir.rb:95:in `mktmpdir'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/container/dmg.rb:34:in `mount'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/container/dmg.rb:18:in `extract_to_dir'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/container/base.rb:18:in `extract'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:171:in `extract_primary_container'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:66:in `stage'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:87:in `install'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:20:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:14:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:14:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/abstract_command.rb:33:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:93:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:157:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:122:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:7:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:87:in `<main>'
Installing licecap has failed!
```

**Comment by vitorgalvao** [comment]
> If I missed the template

You did, because you haven’t filled it and didn’t provide all the requested information. But we’re interested in this bug, so please try again.

**Comment by Taytay** [comment]
Well, I waited too long, and I can no longer reproduce this unfortunately/fortunately, so you are free to close. I wish I'd had more time to investigate. You're just using `mktmpdir`, which cleans up the folder after the block you give it. I can't imagine why this was failing, and wish I'd had time to ls that folder. 
If it helps, I was installing a _lot_ of other stuff at the same time from a large Brewfile. (But then I got the same error repeatedly - this is really odd...)
